commons/src/main/java/org/infinispan/commons/hash/MurmurHash3.java

```diff
@@ -432,28 +432,28 @@ private long MurmurHash3_x64_64_String(String s, long seed) {
          }
 
          if (cp <= 0x7f) {
-            addByte(state, (byte) cp, ++byteLen);
+            addByte(state, (byte) cp, byteLen++);
```

Comment:
No, I mean - I suppose the test passed with commit 1, and it passes with commit 2 while this seems that these should produce different hashcodes. So is there an issue with the tests or is my assumption incorrect?
In the end we could squash them, though.